Task2/Task2/Task2/Infrastructure/JsonReader.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+using Task2.Products;
+
+namespace Task2.Infrastructure
+{
+    public static class JsonReader<T>
+    {
+        public static void JsonReadFromFileBrick(out List<T> listBricks, string path)
```

Comment:
Replase `List<T>` on just `T`.
It's better to return from the method with `return` than as `out`. 
Also rename this method to "ReadFromFile", because you're using a word "Json" in class name.

Reply:
fixed